website/blog/2022-12-14-databend-weekly-72.md

```diff
@@ -0,0 +1,187 @@
+
+**Stage**
+
+- support select from URI ([#9247](https://github.com/datafuselabs/databend/pull/9247))
+
+**Streaming Load**
+
+- support `file_format` syntax in streaming load insert sql ([#9063](https://github.com/datafuselabs/databend/pull/9063))
+
+**Planner**
+
+- push down `limit` to `union` ([#9210](https://github.com/datafuselabs/databend/pull/9210))
+
+**Query**
+
+- use `analyze table` instead of `optimize table statistic` ([#9143](https://github.com/datafuselabs/databend/pull/9143))
+- fast parse insert values ([#9214](https://github.com/datafuselabs/databend/pull/9214))
+
+**Storage**
+
+- use distinct count calculated by the xor hash function ([#9159](https://github.com/datafuselabs/databend/pull/9159))
+- `read_parquet` read meta before read data ([#9154](https://github.com/datafuselabs/databend/pull/9154))
+- push down filter to parquet reader ([#9199](https://github.com/datafuselabs/databend/pull/9199))
+- prune row groups before reading  ([#9228](https://github.com/datafuselabs/databend/pull/9228))
+
+**Open Sharing**
+
+- add prototype open sharing and add sharing stateful tests ([#9177](https://github.com/datafuselabs/databend/pull/9177))
+
+### Code Refactoring :tada:
+
+**\***
+
+- simplify the global data registry logic  ([#9187](https://github.com/datafuselabs/databend/pull/9187))
+
+**Storage**
+
+- refactor deletion ([#8824](https://github.com/datafuselabs/databend/pull/8824))
+
+### Build/Testing/CI Infra Changes :electric_plug:
+
+- release databend deb package and databend with hive ([#9138](https://github.com/datafuselabs/databend/pull/9138), [#9241](https://github.com/datafuselabs/databend/pull/9241), etc.)
+
+### Bug Fixes :wrench:
+
+**Format**
+
+- support ASCII control code hex as format field delimiter ([#9160](https://github.com/datafuselabs/databend/pull/9160))
+
+**Planner**
+
+- prewhere_column empty and predicate is not const will return empty ([#9116](https://github.com/datafuselabs/databend/pull/9116))
+- don't push down topk to Merge when it's child is Aggregate ([#9183](https://github.com/datafuselabs/databend/pull/9183))
+- fix nullable column validity not equal ([#9220](https://github.com/datafuselabs/databend/pull/9220))
+
+**Query**
+
+- address unit test hang on test_insert ([#9242](https://github.com/datafuselabs/databend/pull/9242))
+
+**Storage**
+
+- too many io requests for read blocks during compact ([#9128](https://github.com/datafuselabs/databend/pull/9128))
+- collect orphan snapshots ([#9108](https://github.com/datafuselabs/databend/pull/9108))
+
+## What's On In Databend
+
+Stay connected with the latest news about Databend.
+
+#### Unifying Format Options and Removing `format_*` Settings
+
+> ***Breaking Change*** We plan to unify the formatting options and remove `format_*` settings in the future. 
+> This means that we will **disable** the old format.
+>  Work is in progress and support for `FILE_FORMAT` has now been completed in streming load.
```

Comment:
1. streming -> streaming.
2.  Work is done,  we are ready to disable it,   only a matter of timing.